external_resources/constants.py

```diff
@@ -25,3 +25,18 @@
 # constants for Celery task
 EXTERNAL_RESOURCE_TASK_RATE_LIMIT = "100/s"
 EXTERNAL_RESOURCE_TASK_PRIORITY = 4  # Lowest priority from range (0 - 4)
+
+# constants for user agent
+USER_AGENT_STRING = (
+    "Mozilla/5.0 (Windows NT 10.0; Win64; x64) "
+    "AppleWebKit/537.36 (KHTML, like Gecko) "
+    "Chrome/114.0.0.0 "
+    "Safari/537.36"
+)
+
+USER_AGENT_TIMEOUT = 30
```

Comment:
@pt2302 We have observed this timeout is not enough for some URLs. This should be ~60-90 sec. However, a longer timeout may have a direct impact on link checking. We should consider adding this timeout as an environment variable.

Reply:
Let's address this in a follow-up PR so we can get this merged. 

I'm not sure I understand what you mean by "direct impact on link checking" but we could discuss that in a separate issue.